portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,55 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ryan's Portfolio</title>
     <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <script>
+
+        function addRandomFact() {
+            const facts = 
+                ['hair', 'height', 'lakers', 'hammock', 'ucsb', 'italian'];
```

Comment:
This approach requires that every time you add a new fact, you add it to the HTML, give it an ID, aand then add it to this array.

Instead you can consider using document.getElementsByClassName or querySelectorAll (as Tim mentioned today!) to get all elements with the fact class (you would need to add this to the HTML).  then you can pick a random fact from this array and change the styling for that element.

This would allow you add a many facts as you want to the HTML without having to also change the JS!

Reply:
I like Megan's idea, and have a suggestion on top of it. If you go with the class name `fact`, you can set the CSS style
```
.fact {
  display: none;
}
```
And then add and remove a class called, for instance, `visible`, which is used for a styling rule that looks like:
```
.fact.visible {
  display: block;
}
```
If you're curious, the latter rule's `display` setting will override the former's because it has higher [CSS specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity).

This approach would let you avoid explicitly setting `style="display:none"` on all of the fact elements, and lets you have a visibility class to add and remove (see next comment) rather than having the JS modify the inline styles on each element (doing so is fine for this case, but in a big project, you might want to have the flexibility to use the same class for lots of code and then be able to, say, add a border to all visible elements without having to update all of the different hide/show code.)